api/endpoints/timeline.py

```diff
@@ -26,7 +26,8 @@ def make_blueprint(con):
         "yearly_belonging_rate_W": lambda: yearly_metric.belonging("W"),
         "yearly_belonging_rate_B": lambda: yearly_metric.belonging("B"),
         "yearly_belonging_rate_A": lambda: yearly_metric.belonging("A"),
-        "yearly_belonging_rate_H": lambda: yearly_metric.belonging("H")
+        "yearly_belonging_rate_H": lambda: yearly_metric.belonging("H"),
```

Comment:
Oops, we forgot to commit your changes that you were working on before moving to a new branch. Can you make a branch that has just the changes to the frontend?

Reply:
These changes have been removed and exist on the timeline_disabilities branch

Reply:
Excellent! it looks like you moved the metric implementation, but not the test.

Since the test is still in this branch, it is failing.

Once you move the test, we can merge!

Reply:
Test was removed, all tests pass now.